corehq/pillows/groups_to_user.py

```diff
@@ -67,16 +68,29 @@ def update_es_user_with_groups(group_doc, es_client=None):
             doc = {"__group_ids": list(user_source.group_ids), "__group_names": list(user_source.group_names)}
             es_client.update(USER_INDEX, ES_META['users'].type, user_source.user_id, body={"doc": doc})
 
+    for user_source in stream_user_sources(group_doc.get("removed_users", [])):
+        if group_doc["name"] in user_source.group_names or group_doc["_id"] in user_source.group_ids:
+            user_source.group_ids.remove(group_doc["_id"])
+            user_source.group_names.remove(group_doc["name"])
+            doc = {"__group_ids": list(user_source.group_ids), "__group_names": list(user_source.group_names)}
+            es_client.update(USER_INDEX, ES_META['users'].type, user_source.user_id, body={"doc": doc})
```

Comment:
err sorry, i think it's functionally fine as is. if we want to remove the user from that property it'd require us to do a couch fetch which seems meh. i don't see that property getting too large